packages/react/src/components/Toggle/Toggle.js

```diff
@@ -37,7 +38,7 @@ export function Toggle({
   });
 
   function handleClick(e) {
-    if (!readOnly) {
+    if (!readOnly && !disabled) {
```

Comment:
We probably don't want to fire the `onClick` when disabled, either, since it wouldn't match keyboard parity (can't be focused).

Should we just add `!disabled` to `L87`?

```jsx
if (buttonElement.current && e.target !== buttonElement.current && !disabled) {
```

Reply:
So I tried adding it there but it's still showing the styles, I wonder if it has to do with CSS :disabled state

Reply:
Got it, I'm figuring out the selectors now